jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/ApplyServiceTest.java

```diff
@@ -194,7 +192,7 @@ public void testCreateRouteInServiceOnlyMode() {
         applyService.apply(route, "route.yml");
 
         collector.assertEventsNotRecorded("get-route");
-        assertEquals(1, mockServer.getMockServer().getRequestCount());
+        assertEquals(0, mockServer.getOpenShiftMockServer().getRequestCount());
```

Comment:
This is odd. The Route creation POST request should be recorded, right?

Reply:
Allow create is disabled in the test, Creation is skipped 

https://github.com/eclipse/jkube/blob/7107cf31404be27683d12f74247c931ddd8a9f81/jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/ApplyService.java#L660-L666

Reply:
On master this is the current log:
```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Nov 17, 2021 4:21:06 PM okhttp3.mockwebserver.MockWebServer$2 execute
INFO: MockWebServer[45375] starting to accept connections
Nov 17, 2021 4:21:07 PM okhttp3.mockwebserver.MockWebServer$3 processOneRequest
INFO: MockWebServer[45375] received request: GET /apis HTTP/1.1 and responded: HTTP/1.1 404 Client Error
Nov 17, 2021 4:21:07 PM okhttp3.mockwebserver.MockWebServer$2 acceptConnections
INFO: MockWebServer[45375] done accepting connections: Socket closed
```

Reply:
:+1: